sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1032,3 +1197,276 @@ def _from_generated(cls, score):
     def __repr__(self):
         return "SentimentConfidenceScores(positive={}, neutral={}, negative={})" \
             .format(self.positive, self.neutral, self.negative)[:1024]
+
+
+class EntitiesRecognitionTask(DictMixin):
+    """EntitiesRecognitionTask encapsulates the parameters for starting a long-running Entities Recognition operation.
+
+    :ivar str name: An optional descriptive name for the task.
+    :ivar str model_version: The model version to use for the analysis.
+    :ivar str string_index_type: An optional string for specifying the method used to interpret string offsets.  
+        Defaults to Text Elements (Graphemes) according to Unicode v8.0.0. For additional information see 
+        https://aka.ms/text-analytics-offsets
+    """
+
+    def __init__(self, **kwargs):
+        self.model_version = kwargs.get("model_version", "latest")
```

Comment:
i don't think you're capturing the `name` property

Reply:
Yep, I removed it from the docstring since it turns out this actually isn't in the Swagger definition.